js/components/main.js

```diff
@@ -374,6 +376,11 @@ class Main extends ImmutableComponent {
       windowActions.setContextMenuDetail()
     })
 
+    ipc.on(messages.IMPORTER_LIST, (e, detail) => {
+      windowActions.setImportBrowserDataDetail(detail)
```

Comment:
ideally actions should not be setters, they should describe the actual action. In this case something like `appAction.supportedBrowsersUpdated` which would then be handled in the windowStore and should call helper methods to update the state (don't call actions from actions).